deploy/operator/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I was hoping we could get rid of kustomize entirely. I am fine creating "all-in-one" yaml for deploying it all using http (otherwise `kubectl apply -f deploy/operator/` just works) but foreach and echo can do the same job without a kustomize config